nosecurity/src/main/resources/static/logincss.css

```diff
@@ -0,0 +1,71 @@
+/* Bordered form */
```

Comment:
Why add this file? As discussed last week, the backend should remove all UI related code html, css, javascript etc.